install_config/master_node_configuration.adoc

```diff
@@ -728,9 +731,19 @@ stamp encoded in their filename.
 |`MaximumFileSizeMegabytes`
 |Specifies maximum size in megabytes of the log file before it gets rotated.
 Defaults to 100MB.
-
 |===
 
+
+.Example Audit Configuration
+----
+auditConfig:
+  auditFilePath: "/var/log/audit-ocp.log"
+  enabled: true
+  maximumFileRetentionDays: 10
+  maximumFileSizeMegabytes: 10
+  maximumRetainedFiles: 10
```

Comment:
Dunno if the camelcase for the options here vs the initcap in the above table matters or not.